### PR TITLE
completed fetch-in-react exercise - Evan Hong

### PR DESCRIPTION
**Pull request by EvanS-Hong**
image

### CODE REVIEW

File: fetch-in-react/my-app/src/User.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      try {
+        const res = await fetch(`https://jsonplaceholder.typicode.com/users/${userId}`);
+        const jsonData = await res.json();
+        if (!res.ok) {
+          throw new Error(`error ${res.status}`);
+        } else {
+          setUser(jsonData);
+        }
+      } catch (err) {
+        console.error(err);
```
Comment (RobertGardner) [inline fetch-in-react/my-app/src/User.js]: Set the error state so it will display on the page.

File: fetch-in-react/my-app/src/Users.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          setUsers(jsonData);
+        }
+      } catch (err) {
+        setError(err);
+      } finally {
+        setIsLoading(false);
+      }
+    }
+    listUsers();
+  });
```
Comment (RobertGardner) [inline fetch-in-react/my-app/src/Users.js]: Needs a dependency array.

**Review by RobertGardner — CHANGES_REQUESTED** [review]

File: fetch-in-react/my-app/src/User.js (hunk 1 above)
Comment (EvanS-Hong) [reply fetch-in-react/my-app/src/User.js]: it seems my save didn't push resent just now.

File: fetch-in-react/my-app/src/Users.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ export default function Users() {
       }
     }
     listUsers();
-  });
+  },[users]);
```
Comment (RobertGardner) [inline fetch-in-react/my-app/src/Users.js]: Is this a dependency?
Also, I noticed that `user` isn't used anywhere.

**Review by RobertGardner — CHANGES_REQUESTED** [review]

File: fetch-in-react/my-app/src/Users.js (hunk 3 above)
Comment (RobertGardner) [reply fetch-in-react/my-app/src/Users.js]: Actually, `user` is used. Sorry about that!

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**